src/index.js

```diff
@@ -130,9 +132,10 @@ export default function onClickOutsideHOC(WrappedComponent, config) {
           events = [events];
         }
         events.forEach(eventName => {
-          const handlerOptions = !this.props.preventDefault && ['touchstart', 'touchmove'].indexOf(eventName) !== -1
-            ? { passive: true }
-            : null;
+          const handlerOptions =
```

Comment:
You're going to have to explain this a bit more then, because those three things will be true irrespective of whether you use a ternary here to turn `handlerOptions` into an "object or a null", or whether we use proper code to make sure it's always an object, just with `passive` set appropriately.

In both cases, that event listening options object will need to be available (either because it got cached or because we regenerate it) when `addEventListener` is invoked. Changing the assignment for `handlerOptions` to be an object with `{ passive: false }` rather than a `null` by default should make zero difference in execution with respect to addEventListener/removeEventListener functionality